specs/signature-specification.md

```diff
@@ -40,21 +40,65 @@ Signature Manifest Example
 ```
 
 - **`artifactType`** (*string*): This REQUIRED property references the Notary version of the signature: `application/vnd.cncf.notary.signature`.
-- **`blobs`** (*array of objects*): This REQUIRED property contains collection of only one [artifact descriptor][artifact-descriptor] referencing signature envelope.
+- **`blobs`** (*array of objects*): This REQUIRED property contains collection of only one [OCI descriptor][oci-descriptor] referencing signature envelope.
   - **`mediaType`** (*string*): This REQUIRED property contains media type of signature envelope blob. Following values are supported
     - `application/jose+json`
-- **`subject`** (*descriptor*): A REQUIRED artifact descriptor referencing the signed manifest, including, but not limited to image manifest, image index, oras-artifact manifest.
+    - `application/cose`
+- **`subject`** (*descriptor*): A REQUIRED artifact descriptor referencing the signed manifest, including, but not limited to image manifest, image index, OCI artifact manifest.
 - **`annotations`** (*string-string map*): This REQUIRED property contains metadata for the artifact manifest.
   It is being used to store information about the signature.
   Keys using the `io.cncf.notary` namespace are reserved for use in Notary and MUST NOT be used by other specifications.
   - **`io.cncf.notary.x509chain.thumbprint#S256`**: A REQUIRED annotation whose value contains the list of SHA-256 fingerprint of signing certificate and certificate chain (including root) used for signature generation. The annotation name contains the hash algorithm as a suffix (`#S256`) and can be extended to support other hashing algorithms in future.
     The list of fingerprints is present as a JSON array string, corresponding to ordered certificates in [*Certificate Chain* unsigned attribute](#unsigned-attributes) in the signature envelope.
 
+Notary v2 supports using [OCI image manifest][oci-image-manifest] to store the signature in the repository that doesn't support [OCI artifact manifest][oci-artifact-manifest].
```

Comment:
that doesn't support OCI artifact manifest or listing-referrers API? 
Also how will we decide Notary v2 supports using OCI image manifest ?

Reply:
* This spec is about signature format, so only OCI artifact manifest is considered.
* Notary v2 should always use OCI artifact manifest to store the signature in the repository. however, not all the repositories implementing the latest image spec 1.1.0, it takes some time, and there is no deadline, so Notary v2 needs to consider how to store signature in the repository that doesn't support OCI artifact manifest. Use OCI image manifest is the solution. There is not standard way to do it. Currently it is based on the response code 400 and error code (manifest invalid or un-supported) after pushing OCI artifact manifest